api/src/wfl/service/postgres.clj

```diff
@@ -42,6 +42,15 @@
     (jdbc/query tx (format "SELECT * FROM %s" table))
     (throw (ex-info (format "Table %s does not exist" table) {:cause "no-such-table"}))))
 
+(defn ^:private cromwell-status
+  "`status` of the workflow with `uuid` in `cromwell`."
+  [cromwell uuid]
+  (-> (str/join "/" [cromwell "api" "workflows" "v1" uuid "status"])
+      (http/get {:headers (once/get-auth-header)})
+      :body
+      util/parse-json
+      :status))
```

Comment:
Consolidate this cromwell-status with the one in workloads.clj?

Reply:
Good spot! That one isn't used so I'll just remove it!